components/NewsAndUpdatesModal.js

```diff
@@ -16,9 +17,28 @@ import MessageBox from './MessageBox';
 import StyledButton from './StyledButton';
 import StyledCarousel from './StyledCarousel';
 import StyledLink from './StyledLink';
-import Modal, { ModalBody, ModalFooter, ModalHeader } from './StyledModal';
+import Modal, { CloseIcon, ModalBody, ModalFooter, ModalHeader } from './StyledModal';
 import { P, Span } from './Text';
 
+const ModalHeaderWrapper = styled(ModalHeader)`
+  height: 58px;
+  ${CloseIcon} {
+    margin-top: 30px;
+    height: 20px;
+    width: 20px;
```

Comment:
Looks like it's a different size on Figma

![image](https://user-images.githubusercontent.com/1556356/125042817-925bb080-e09a-11eb-998d-5d48c81f2741.png)


```suggestion
    height: 12px;
    width: 12px;
```

Reply:
Thanks for pointing it out. Corrected it. 😄